modules/scheduling-a-pipeline-run.adoc

```diff
@@ -19,31 +18,29 @@ endif::[]
-. From the *Pipeline version* list, select the pipeline version to create a run for. Alternatively, to upload a new version, click *Upload new version* and complete the relevant fields in the *Upload new version* dialog.
-. Configure the run type by performing one of the following sets of actions:
-* Select *Run once immediately after creation* to specify the run executes once, and immediately after its creation.
-* Select *Schedule recurring run* to schedule the run to recur.
-... Configure the run's trigger type.
-.... Select *Periodic* to specify an execution frequency. Enter a numerical value and select an execution frequency from the list.
-.... Select *Cron* to specify the execution schedule in `cron` format. This creates a cron job to execute the run. Click the *Copy* button (image:images/osd-copy.png[]) to copy the cron job schedule to the clipboard. The field furthest to the left represents seconds. For more information about scheduling tasks using the supported `cron` format, see link:https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format[Cron Expression Format].
-... Configure the run's duration.
-.... Select the *Start date* check box to specify a start date for the run. Select the run's start date using the *Calendar* and the start time from the list of times.
-.... Select the *End date* check box to specify an end date for the run. Select the run's end date using the *Calendar* and the end time from the list of times.
-. Configure the input parameters for the run by selecting the parameters from the list.
-. Click *Create*.
+The *Runs* page opens.
+. From the *Project* list, select the project that you want to schedule a run for.
+. Click the *Schedules* tab.
+. Click *Schedule run*. 
+. On the *Schedule run* page, configure the run that you are scheduling:
+.. In the *Name* field, enter a name for the run.
+.. In the *Description* field, enter a description for the run.
+.. From the *Trigger type* list, select one of the following options:
+* Select *Periodic* to specify an execution frequency. In the *Run every* field, enter a numerical value and select an execution frequency from the list.
+* Select *Cron* to specify the execution schedule in `cron` format. This creates a cron job to execute the run. Click the *Copy* button (image:images/osd-copy.png[]) to copy the cron job schedule to the clipboard. The field furthest to the left represents seconds. For more information about scheduling tasks using the supported `cron` format, see link:https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format[Cron Expression Format].
+.. In the *Maximum concurrent runs* field, specify the number of runs that can execute concurrently, from a range of one to ten.  
+.. For *Start date*, specify a start date for the run. Select a start date using the *Calendar* and the start time from the list of times.
```

Comment:
same comment as previous applies here to lines 34 and 35
by using the Calendar. Select the start time...